pkgs/misc/emulators/mednafen/default.nix

```diff
@@ -1,36 +1,30 @@
-{ stdenv, fetchurl, pkgconfig
-, libX11, mesa, freeglut
-, libjack2, libcdio, libsndfile, libsamplerate
-, SDL, SDL_net, zlib
-}:
+{ stdenv, fetchurl, pkgconfig, libX11, mesa, freeglut, libjack2,
+, libcdio, libsndfile, libsamplerate, SDL, SDL_net, zlib }:
```

Comment:
Syntax error here

Reply:
Made a last-minute change, sorry about that. Fixed now.